command/agent/agent_endpoint_test.go

```diff
@@ -481,11 +521,17 @@ func TestAgent_PprofRequest(t *testing.T) {
 		addNodeID   bool
 		addServerID bool
 		expectedErr string
+		nilServer   bool
```

Comment:
Maybe this is more appropriate as `clientOnly`? or `serverDisabled` as it conveys the concept better?